### ISSUE TITLE
[New addon] image uploader

### ISSUE THREAD

**Issue by JeffaloBob** [ScratchAddons/ScratchAddons#41]
**Describe the new addon**
it uploads images

**Link to the new addon**
https://github.com/jeffaloBob/scratch-image-uploader

it comes in userscript/extension version, (although the userscript is a super hacky way of doing it...) i'm not really sure how this whole project works, but i'm assuming porting my thingy can't be too hard and its super cool so it must be there...

**Comment by WorldLanguages** [comment]
Nice. They're just 3 content scripts so they should work. I see you're loading JQuery tho D: Luckily it's not necessary sinse userscripts run in the context of the Scratch page, not the extension content script, so it should be fine as long as the JQuery version matches.

**Comment by JeffaloBob** [comment]
yeah i needed jquery for the image uploading... it should work on scratch since i stole all the jquery code from your bookmarklet anyways
